packages/esm-patient-chart-app/src/index.ts

```diff
@@ -44,6 +44,7 @@ function setupOpenMRS() {
     extensions: [
       {
         id: "patient-chart-nav-items",
+        slot: "nav-menu-slot",
```

Comment:
This should be attached dynamically. We only want to have this attached when the patient chart is rendered.

Reply:
oh okay, I was already wondering why this was missing.
Well the attach mechanism isn't working for me – I'll have look why.